DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using LT.DigitalOffice.CheckRightsService.Commands.Interfaces;
-using System.Collections.Generic;
+using LT.DigitalOffice.CheckRightsService.Models;
 using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
 using Microsoft.AspNetCore.Mvc;
-using LT.DigitalOffice.CheckRightsService.Models;
+using System.Collections.Generic;
```

Comment:
First connect system namespaces and than namespaces created in the our project

Reply:
@djino97  said that it was more convenient for him.
I leave it like this for now because this is the default behavior.